### PR TITLE
fixes test/integration/ttlcontroller staticcheck

### PR DESCRIPTION
**Pull request by ghost**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Addresses a static check issue ( [SA2002](https://staticcheck.io/docs/checks#SA2002) ) for `test/integration/ttlcontroller`
**Which issue(s) this PR fixes**:

Part of #92402

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by a1k24** [comment]
I've signed the CLA

File: test/integration/ttlcontroller/ttlcontroller_test.go [hunk 1]
```diff
@@ -60,27 +61,48 @@ func createNodes(t *testing.T, client *clientset.Clientset, startIndex, endIndex
...
 		}(i)
 	}
-	wg.Wait()
+
+	go func() { // wait in another go-routine to close channel
+		wg.Wait()
+		close(errs)
+	}()
+
+	for err := range errs {
```
Comment (oomichi) [inline test/integration/ttlcontroller/ttlcontroller_test.go:76]: Don't we need to add `if err != nil {` like line 102?

Comment (a1k24) [reply test/integration/ttlcontroller/ttlcontroller_test.go:76]: Yes, thanks for pointing out. Let me make that change.

Comment (a1k24) [reply test/integration/ttlcontroller/ttlcontroller_test.go:76]: Have changed that in the latest commit.

**Comment by oomichi** [comment]
Thanks for updating.

/lgtm

**Comment by a1k24** [comment]
Thank you @oomichi ! Can you please approve the PR if everything looks fine?

**Comment by oomichi** [comment]
for `/test`

/approve

**Pull request merged**